test/llvm-intrinsics/tan.ll

```diff
@@ -0,0 +1,77 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck %s
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+
+target datalayout = "e-p:32:32-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir"
+
+; CHECK: ExtInstImport [[ExtInstSetId:[0-9]+]] "OpenCL.std"
```

Comment:
I'm fine with this notation, but just FYI for numerical variables for LIT regexp the following is considered a bit better: [[#ExtInstSetId:]] and use of the var will be [[#ExtInstSetId]]

Reply:
+1 To use numerical variables through all the test.

Reply:
Done, thanks both for the suggestion.